### ISSUE TITLE
[Feature Request] Change language of info of the movies/tvshows

### ISSUE THREAD

**Issue by drinfernoo** [a4k-openproject/plugin.video.openmeta#6]
Requested by @lidorx3:

Hello i can not change the language of TMDB shown lists of movies/tvshows to my own language which is set up as my kodi language and also TMDB has this language..in other meta forks or even on meta itself it was possible than i would really apperiacte if you could help me set it up to my language on this meta fork too..
Thanks at advance

**Comment by kodishu** [comment]
I'm waiting for the same. Can't change movie/tv Shows descriptions to my native language.

**Comment by PrevotYann** [comment]
Same as a French dude, I'd like to have descriptions in french :)

**Comment by drinfernoo** [comment]
@kodishu, @PrevotYann, @noahf1, would you be able to test 1.4.10 and see if these are satisfactory changes?

**Comment by cbec-dev** [comment]
@drinfernoo I tested it, it seems to work for movie posters for me but not for titles or description, it also doesn't seem to work for tv shows but I haven't tested those that much, I'm using spanish btw.

**Comment by noahrmal** [comment]
It does not work for me as well. Titles and descriptions are still in english instead of my kodi language (german).

**Comment by melmel0101** [comment]
May be you can unchecked the "enable library update" in open meta parameter and after add the movies and tvshow folders directly in Kodi parameters library.
When you add new folder in library you can choose the language of info in (information provider parameter).

**Comment by noahrmal** [comment]
> 
> 
> May be you can unchecked the "enable library update" in open meta parameter and after add the movies and tvshow folders directly in Kodi parameters library.
> When you add new folder in library you can choose the language of info in (information provider parameter).

Yes this works, but the language for titles and descriptions of movies/episodes within the open meta addon are still in english e.g. when exploring new movies/shows ("Popular", "Most watched", "Trending" etc.). I tried to implement it in a quick and dirty way (#45) and then the titles/descriptions of media in the trakt/tmdb lists are displayed in the selected language.
